ecommerce/migrations/0003_company_couponorder.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 2.1.7 on 2019-04-03 19:30
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("ecommerce", "0002_coupon_additional_fields")]
+
+    operations = [
+        migrations.CreateModel(
+            name="Company",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("created_on", models.DateTimeField(auto_now_add=True)),
+                ("updated_on", models.DateTimeField(auto_now=True)),
+                ("name", models.CharField(max_length=512, unique=True)),
+            ],
+            options={"abstract": False},
+        ),
+        migrations.CreateModel(
+            name="CouponOrder",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("created_on", models.DateTimeField(auto_now_add=True)),
+                ("updated_on", models.DateTimeField(auto_now=True)),
+                (
+                    "payment_type",
+                    models.CharField(
+                        blank=True,
+                        choices=[
+                            ("credit_card", "credit_card"),
+                            ("payment_order", "payment_order"),
+                        ],
+                        max_length=128,
+                        null=True,
+                    ),
+                ),
+                (
+                    "order_number",
```

Comment:
"payment_id" so it's not confused with orders in our system ?

Reply:
👍